README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Worked on descriptive stats with pandas data frames:

head()shows first few rows
shape() shows the dimenions and scale
dtypes property describes data types of each column
describe() shows mean, stdev and distributions (25, 50, 75th percentiles)
corr() shows pairwise correlation (using pearson correlation by default)

I wrote a pearson correlation function to get a little practice in python and to bone up on these types of stats usages:
```
def r(data, colx, coly):
        meanx = data.describe()[colx]["mean"]
	meany = data.describe()[coly]["mean"]
	stdx = data.describe()[colx]["std"]
	stdy = data.describe()[coly]["std"]
	n = data.shape[0]
	s = 0
	for ix, row in data.iterrows():
		cx = ((row[colx] - meanx) / stdx)
		cy = ((row[coly] - meany) / stdy)
		s = s + (cx) * (cy)
	r = s * (float(1)/(n-1))
	return r
```

pariwise correlation is -1 to 1, and is used to determine whether two random variables are 'correlated' (i.e. move in relation) to each other. 1 is perfect correlation, 0 is uncorrelated, -1 is perfect inverse correlation (move perfectly in opposite directions).